packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -66,7 +67,7 @@ const logAndFormatError = (err) => {
         return {message: err.message, status: err.status, stack: err.stack}
     } else {
         const cause = err.stack || err.toString()
-        console.error(cause)
+        logger.error(cause, {key:'pwa-kit-react-sdk', details:['render']})
```

Comment:
There are other `console.error` and `console.warn` used in multiple packages. Do we plan to convert them to use `logger` too?

Reply:
Yes, this is a TODO that will be addressed in this PR once we align in the logger interface. I just replaced a few console statements as an example.